packages/frontend/src/lib/RecipeDetails.svelte

```diff
@@ -20,6 +22,9 @@ $: envState = $environmentStates.find((env: EnvironmentState) => env.recipeId ==
 $: recipe = $catalog.recipes.find(r => r.id === recipeId);
 $: recipeStatus = $recipes.get(recipeId);
 $: model = $catalog.models.find(m => m.id === modelId);
+$: localPath = $localRepositories.find(
+  local => !!local.labels && 'recipe-id' in local.labels && local.labels['recipe-id'] === recipeId,
```

Comment:
Such logic should be in the store or utility to be reused

Reply:
Good idea, I will create an utility class